pkg/cluster/docker_for_mac_test.go

```diff
@@ -0,0 +1,126 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNoOp(t *testing.T) {
+	f := newD4MFixture(t)
+	defer f.TearDown()
+
+	ctx := context.Background()
+	settings, err := f.d4m.settings(ctx)
+	require.NoError(t, err)
+
+	err = f.d4m.writeSettings(ctx, settings)
+	require.NoError(t, err)
+
+	assert.Equal(t,
+		f.postSettings,
+		f.readerToMap(strings.NewReader(postSettingsJSON)))
+}
+
+func TestEnableKubernetes(t *testing.T) {
+	f := newD4MFixture(t)
+	defer f.TearDown()
+
+	ctx := context.Background()
+	settings, err := f.d4m.settings(ctx)
+	require.NoError(t, err)
+
+	changed, err := f.d4m.ensureK8sEnabled(settings)
+	assert.True(t, changed)
```

Comment:
why not `require` here?

Reply:
assert allows the test to continue executing, so we can see the rest of the output! it's not a big thing but it's a nice feature of the library

Reply:
ah sorry, i meant what is there in the rest of the output that we care about if we've failed here? But, nbd.